src/IB/IBFEMethod.cpp

```diff
@@ -2966,6 +2966,7 @@ IBFEMethod::getFromInput(const Pointer<Database>& db, bool /*is_from_restart*/)
         d_default_interp_spec.quad_order = FIRST;
         d_default_interp_spec.use_adaptive_quadrature = false;
         d_default_interp_spec.use_consistent_mass_matrix = false;
+        d_default_interp_spec.allow_rules_with_negative_weights = false;
```

Comment:
Since this changes the default behavior I would prefer to wait on merging this until after the release.

Reply:
It does not change the default behavior because the nodal rules only have positive weights.

Reply:
oh, I see - this is for `nodal_quadrature`. I should have scrolled up.

Reply:
Most of these settings do not have any impact for the specialized nodal interaction code path.